app/src/main/java/org/hackillinois/android/view/project/ProjectInfoFragment.kt

```diff
@@ -27,6 +27,11 @@ class ProjectInfoFragment : Fragment() {
         viewModel = ViewModelProviders.of(this).get(ProjectInfoViewModel::class.java)
         viewModel.init(projectId)
         viewModel.project.observe(this, Observer { updateProjectUI(it) })
+
+        viewModel.projectFavorited.observe(this, Observer { updateFavoritedUI(it) })
+        starButton.setOnClickListener {
```

Comment:
Also can you move the starButton line to onCreateView and do view.startButton...

Since the view is not created yet, this button is null

Reply:
Yeah not sure why I put it outside onCreateView whoops